src/core/utilities/point/point.h

```diff
@@ -126,6 +126,42 @@ namespace core
         }
 
 
+
+        auto operator+(Type value) const
```

Comment:
why not Type const& ?

Reply:
box.h is similar, but both could be changed